components/pkg-export-docker/src/cli.rs

```diff
@@ -243,6 +243,26 @@ impl<'a, 'b> Cli<'a, 'b> {
 
         Cli { app: app }
     }
+
+    pub fn add_pkg_ident_arg(self, multiple: bool) -> Self {
```

Comment:
Booleans stink. I'd prefer using an enum I think.

Reply:
When you know that you are not going to have more than 2 options ever, booleans is the right way to go IMO. The underlying clap API is using boolean too and this is just being passed verbatim. Don't see the point to introducing an enum, just to be translated to a boolean.

Reply:
I think it's rather for clarity when invoking the function, see [my comment](https://github.com/habitat-sh/habitat/pull/4298#discussion_r156634425).

Reply:
hmm.. OK. I'll change that but i think I'll go with two separate public methods instead.